pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -2539,6 +2540,7 @@ extra-packages:
   - patience ^>= 0.1                    # required by chell-0.4.x
   - persistent >=2.5 && <2.8            # pre-lts-11.x versions neeed by git-annex 6.20180227
   - persistent-sqlite < 2.7             # pre-lts-11.x versions neeed by git-annex 6.20180227
+  - prettyprinter == 1.6.1              # required by ghc 8.8.x, and dhall-1.29.0
```

Comment:
Is that not the latest version anyway?

Reply:
`prettyprinter-1.6.1` is the latest version on Hackage.

However, we frequently(?) get evaluation errors when we rely on packages like `prettyprinter_1_6_0` existing.

Adding `prettyprinter` to the list of `extra-packages` will hopefully help to keep the `haskell-updates` branch from not being able to be evaluated.

The downside of this is that when `prettyprinter-1.6.2` comes out, it won't be as obvious when we need to update all the packages that are currently using `prettyprinter_1_6_1`.

-------------------------------

However, I think you're probably the one that fixes like 90 to 95% of these types of breakages, so I'd be happy to leave it up to you how you want to handle this in the future.